x-pack/plugins/ml/public/application/model_management/models_list.tsx

```diff
@@ -347,19 +355,20 @@ export const ModelsList: FC<Props> = ({
       ),
       'data-test-subj': 'mlModelsTableColumnType',
     },
-    {
-      name: i18n.translate('xpack.ml.trainedModels.modelsList.stateHeader', {
-        defaultMessage: 'State',
-      }),
-      sortable: (item) => item.stats?.deployment_stats?.state,
-      align: 'left',
-      truncateText: true,
-      render: (model: ModelItem) => {
-        const state = model.stats?.deployment_stats?.state;
-        return state ? <EuiBadge color="hollow">{state}</EuiBadge> : null;
-      },
-      'data-test-subj': 'mlModelsTableColumnDeploymentState',
-    },
+    // FIXME show combined deploymnetss state
```

Comment:
Does this need to be re-enabled or address in a follow-up?

Reply:
Added in https://github.com/elastic/kibana/pull/155375/commits/f3b46b3ba27d4551ace90c28f4e45479f3c3a57a